code/modules/reagents/chemistry/reagents.dm

```diff
@@ -97,6 +97,15 @@
 /datum/reagent/proc/on_mob_death(mob/living/M)	//use this to have chems have a "death-triggered" effect
 	return
 
+/datum/reagent/proc/fire_flash_log() //Call this whenever a chemical makes a fire flash
```

Comment:
Can you make this a global proc which takes the `holder` as param? Since the other version of this on `chemical_reaction` does the same and maintaining both will be a pain.

Reply:
Done!